### DISCUSSION TITLE
Summary Footer

### DISCUSSION THREAD

**Discussion by dnjsgur0629** [KevinVandy/material-react-table#235] [Ideas]
Thanks for providing this library.
How about adding a summary footer feature?

like this:
![image](https://user-images.githubusercontent.com/65391273/206094512-feea42db-01c1-44ff-af7b-2e3c50e5d5f2.png)

If this feature is added, I will use it happily for summing a column and displaying it in the footer.
[I found this example here](https://nhn.github.io/tui.grid/latest/tutorial-example09-summary)

**Comment by KevinVandy** [comment]
You just need to look through the docs site a bit more. That feature is displayed here. Use the `enableStickyFooter` prop. https://www.material-react-table.com/docs/examples/aggregation-and-grouping

**Reply by dnjsgur0629** [reply]
Thank you for your help.
